guides/common/modules/proc_installing-vmware-plugin.adoc

```diff
@@ -11,4 +11,6 @@ This allows you to manage and deploy hosts to VMware.
 ----
 # {foreman-installer} --enable-foreman-compute-vmware
 ----
-. Optional: In the {ProjectWebUI}, navigate to *Administer > About* and select the _compute resources_ tab to verify the installation of the VMware plugin.
+
+.Verification
+* In the {ProjectWebUI}, navigate to *Administer > About* and select the _Available Providers_ tab to verify the installation of the VMware plugin.
```

Comment:
```suggestion
* In the {ProjectWebUI}, navigate to *Administer > About* and under *System Status*, select the _Available Providers_ tab to verify the installation of the VMware plugin.
```

Reply:
```suggestion
* In the {ProjectWebUI}, navigate to *Administer* > *About*.
* Under *System Status*, select the *Available Providers* tab to verify the installation of the VMware plugin.
```

Building upon Brian's suggestions, I think we should split these sentences.